lib/en.js

```diff
@@ -32,10 +32,23 @@ const COMMON_DIACRITICS = new FunctionRule((s1, s2) => {
   return [normalize(s1prime, 'NFC'), normalize(s2prime, 'NFC')];
 });
 
+const INFINITIVE_VERBS = new FunctionRule((s1, s2) => {
+  let s1prime = normalize(s1, 'NFD');
+  let s2prime = normalize(s2, 'NFD');
+
+  const RE_BEGINS_WITH_TO = /^[\s]*(to)[\s]*/i;
```

Comment:
when I see a capturing group like `(to)`, I normally expect it to be used in some way, but here it's not, so I'd probably drop the parentheses